### ISSUE TITLE
Stop showing 'loading' before the user enters any data.

### ISSUE THREAD

**Issue by arvind0598** [arvind0598/better-first-issues#9]
the useRepoData hook has these two lines:

```typescript
const { data, error } = useSWR(['/api/search/', repo], repo ? fetcher : null);

isLoading: !error && !data,
```

This ends up meaning that isLoading is true BEFORE the user actually enters any data. This issue should enhance this and stop showing the "Loading" text if that's the case.

**Comment by GuScarpim** [comment]
I just fixed this:
https://github.com/arvind0598/better-first-issues/pull/15

![Uploading chrome-capture.gif…]()
s

**Comment by arvind0598** [comment]
Fixed in #15
